extensions/applicationinsights-react-native/src/ReactNativePlugin.ts

```diff
@@ -75,6 +98,24 @@ export class ReactNativePlugin implements ITelemetryPlugin {
         this._device.deviceClass = newType;
     }
 
+    trackMetric(metric: IMetricTelemetry, customProperties: ICustomProperties) {
```

Comment:
private _trackMetric

Reply:
Just realized trackMetric is not used yet, removed it for now.